docs/developers/specification.md

```diff
@@ -125,6 +125,7 @@ break
 continue
 else
 false
+pub
```

Comment:
Could you sort this alphabetically?

Reply:
This list was not sorted alphabetically before, so I thought this should not be the case

Reply:
Is it not?